src/transform/src/analysis/equivalences.rs

```diff
@@ -508,6 +508,8 @@ impl EquivalenceClasses {
             }
         }
 
+        self.minimize(&None);
```

Comment:
Do you mind adding the explanation from the PR here as a comment?

Reply:
Done (sort of; comment added and statement relocated).